django_sage_meta/admin/insight.py

```diff
@@ -28,3 +31,25 @@ class InsightAdmin(admin.ModelAdmin):
             },
         ),
     )
+
+    def get_urls(self):
+        urls = super().get_urls()
+        custom_urls = [
+            path(
+                "sync-insights/",
+                self.admin_site.admin_view(self.sync_insights),
+                name="sync_insights",
+            )
+        ]
+        return custom_urls + urls
+
+    def sync_insights(self, request):
+        try:
+            SyncService.sync_insights()
+            self.message_user(
+                request, _("Instagram insights synchronized successfully.")
+            )
+            return HttpResponseRedirect("/admin/django_sage_meta/insight/")
```

Comment:
this way of redirecting is wrong. URL must be found automatically. It is also hard coded.